locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,72 @@
+# Strings for the breach details checklists
+
+## Prompts the user for changes when there is a breach detected of password
+
+# NOTE: { $companyWebsite } is a placeholder for the URL to the website of the company where the breach occurred 
```

Comment:
Is `{ $companyWebsite }` too generic?  Would something like `{ $breachedCompanyUrl }` and `{ $breachedCompanyName }` be more appropriate when thinking about future features and translations?  For example, a data broker might also technically fall under the `{ $companyWebsite }` var.

Reply:
Done thanks!